pkgs/applications/misc/tandoor-recipes/default.nix

```diff
@@ -0,0 +1,139 @@
+{ lib
+, nixosTests
+, fetchFromGitHub
+, fetchpatch
+, callPackage
+, python3
+}:
+let
+  python = python3.override {
+    packageOverrides = self: super: {
+      django = super.django_4;
+
+      # Tests are incompatible with Django 4
+      django-js-reverse = super.django-js-reverse.overridePythonAttrs (_: {
+        doCheck = false;
+      });
+    };
+  };
+
+  common = callPackage ./common.nix { };
+
+  frontend = callPackage ./frontend.nix { };
+in
+python.pkgs.pythonPackages.buildPythonPackage rec {
+  pname = "tandoor-recipes";
+
+  inherit (common) version src;
+
+  format = "other";
+
+  propagatedBuildInputs = with python.pkgs; [
+    beautifulsoup4
+    bleach
+    bleach-allowlist
+    boto3
+    cryptography
+    django
+    django-allauth
+    django-annoying
+    django-auth-ldap
+    django-autocomplete-light
+    django-cleanup
+    django-cors-headers
+    django-crispy-forms
```

Comment:
```suggestion
    django-crispy-forms
    django-debug-toolbar
```

because

```
python3.10-tandoor-recipes> building
python3.10-tandoor-recipes> Traceback (most recent call last):
python3.10-tandoor-recipes>   File "/build/source/manage.py", line 15, in <module>
python3.10-tandoor-recipes>     execute_from_command_line(sys.argv)
python3.10-tandoor-recipes>   File "/nix/store/jqnsvh77gcd4i1860y5mb96chml9v2wa-python3.10-Django-4.1.1/lib/python3.10/site-packages/django/core/management/__init__.py", line 446, in execute_from_command_line
python3.10-tandoor-recipes>     utility.execute()
python3.10-tandoor-recipes>   File "/nix/store/jqnsvh77gcd4i1860y5mb96chml9v2wa-python3.10-Django-4.1.1/lib/python3.10/site-packages/django/core/management/__init__.py", line 420, in execute
python3.10-tandoor-recipes>     django.setup()
python3.10-tandoor-recipes>   File "/nix/store/jqnsvh77gcd4i1860y5mb96chml9v2wa-python3.10-Django-4.1.1/lib/python3.10/site-packages/django/__init__.py", line 24, in setup
python3.10-tandoor-recipes>     apps.populate(settings.INSTALLED_APPS)
python3.10-tandoor-recipes>   File "/nix/store/jqnsvh77gcd4i1860y5mb96chml9v2wa-python3.10-Django-4.1.1/lib/python3.10/site-packages/django/apps/registry.py", line 91, in populate
python3.10-tandoor-recipes>     app_config = AppConfig.create(entry)
python3.10-tandoor-recipes>   File "/nix/store/jqnsvh77gcd4i1860y5mb96chml9v2wa-python3.10-Django-4.1.1/lib/python3.10/site-packages/django/apps/config.py", line 193, in create
python3.10-tandoor-recipes>     import_module(entry)
python3.10-tandoor-recipes>   File "/nix/store/sz0j8k8ljh7y8qgyfxgqb3ws11bcy4gs-python3-3.10.6/lib/python3.10/importlib/__init__.py", line 126, in import_module
python3.10-tandoor-recipes>     return _bootstrap._gcd_import(name[level:], package, level)
python3.10-tandoor-recipes>   File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
python3.10-tandoor-recipes>   File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
python3.10-tandoor-recipes>   File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
python3.10-tandoor-recipes> ModuleNotFoundError: No module named 'debug_toolbar'

Reply:
I fixed the issue, needed to export `DEBUG=0` during `buildPhase`.

Reply:
I think adding a debug-only dependency for an end-user deployment makes little sense, which I why I prefer that fix.

Reply:
Agreed